lib/tests/streamlit/layouts_test.py

```diff
@@ -146,6 +146,80 @@ def test_just_label(self):
         self.assertEqual(expander_block.add_block.expandable.expanded, False)
 
 
+class StatusContainerTest(DeltaGeneratorTestCase):
```

Comment:
Should likely have a test where it throws a `StreamlitAPIException`

Reply:
You mean the `StreamlitAPIException` when the state is unknown?

Reply:
Yep when state is unknown

Reply:
oki :) added `test_throws_error_on_wrong_state`